app/enterprise/0.35-x/kong-manager/authentication/super-admin.md

```diff
@@ -29,4 +29,18 @@ click the email address of the user in order to view them.
 
 5. Click "Generate Registration Link". 
 
-6. Copy the link for later use after completing the account setup.
\ No newline at end of file
+6. Copy the link for later use after completing the account setup.
+
+## How to Create Your First Super Admin Account Post Installation
+
+In the event that the default `kong_admin`, **Super Admin**, was not seeded during the initial database preparation step as defined in [Step 1 in How To Start Kong Enterprise Securely](../../getting-started/start-kong/#step-1), the following steps outline how to create and enable a new Super Admin post installation. 
+
+1. Follow the instructions outlined above to create a new **Super Admin** user account and generate a registration link.
+
+2. Before the link generated above can be used, RBAC and GUI Authentication must be enabled. Follow the instructions on [how to enable Basic Auth on Kong Manager](../basic).
```

Comment:
Not a 404, but will change in order to keep consistent even if we change the path in the future:

```suggestion
2. Before the link generated above can be used, RBAC and GUI Authentication must be enabled. Follow the instructions on [how to enable Basic Auth on Kong Manager](/enterprise/{{page.kong_version}}/kong-manager/authentication/basic).
```